### PR TITLE
WIP: Feature #2

### PR DESCRIPTION
**Pull request by vincentgierisch**
I have implemented a basic game structure, as well as the logic for a move.
Please take a look at it and let's discuss whether we should take that, or make some changes.

I think we also should update the docu with that PR (issue #4 is a blocker).

### CODE REVIEW

File: include/GameHandler.hpp [hunk 1]
```diff
@@ -3,10 +3,13 @@
 
 #include <string>
 #include "network/DataHandlingService.hpp"
+#include "factory/GameFactory.hpp"
+#include <stdint.h>
```
Comment (Jibanizer) [inline include/GameHandler.hpp]: Why is <stdint.h> needed in this file?

Comment (vincentgierisch) [reply include/GameHandler.hpp]: good catch. I need to check it if it's really necessary

Comment (vincentgierisch) [reply include/GameHandler.hpp]: ok, this include was unnecessary

**Review by Jibanizer — APPROVED** [review]
Further documentation is required in the future.

**Pull request merged**